src/magic.cpp

```diff
@@ -1051,6 +1051,37 @@ std::string spell::enumerate_targets() const
     return ret;
 }
 
+std::string spell::list_targeted_monster_ids() const
```

Comment:
Why not name this function `list_targeted_monster_names`? The current name gives the impression that the function returns a monster's string id.